src/api/Services/Cache.cs

```diff
@@ -57,6 +57,7 @@ internal IList<StorageAccountAndContainers> StorageAccounts(string principalId,
 	/// <returns>Cached value or new value if not cached.</returns>
 	internal StorageAccountProperties GetStorageAccountProperties()
 	{
+		if (String.IsNullOrWhiteSpace(Configuration.StorageAccountPropertiesCacheKey)) { return null; }

```

Comment:
This will work to prevent `NullReferenceExceptions` currently, but I think the check really belongs in `ResourceGraphOperations.GetAccountResourceTagValue` because that's where the `tagName` argument is used as an indexer for the return value of the resource graph call.